FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/ActionBarLayoutActivity.kt

```diff
@@ -17,10 +18,19 @@ class ActionBarLayoutActivity: DemoActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        if(isAccessibilityEnabled){
+            label_actionbar_position.isFocusable = true
+            label_actionbar_type.isFocusable = true
+        }
+        else {
+            label_actionbar_position.isFocusable = false
+            label_actionbar_type.isFocusable = false
+        }
+
```

Comment:
@aishwarya56 
I feel mixing of keyboard and accessibility is something across few issues I see.And I don't feel they should be related , Like navigating accessibility control with tab is not a situation , Do you feel these issues should be pushed back ?

Reply:
@mnkshukl ,
This check makes sense in some of the cases. Like this AB#9423, here the click on label works only in accessible mode and hence it needs to be announced only when accessibility is enabled and hence the check was required.

But here, the bug is different and it does not require this check.
So, to sum it up, the fix is not correct and it should announce the radio group name along the radio label.

Announcement should be something like,  checked/unchecked "radio label" button in "radio group"  ......
Hence, we do not need this check and the label need not be in focus.

I hope, this makes sense?

Reply:
@aishwarya56 
I thought adding the "radio group" to the announcements of each of the radio button would make the announcement unnecessarily longer, thats why resorted to just making the labels focusable so that they are announced before the radio buttons.